packages/discord.js/typings/index.d.ts

```diff
@@ -1054,7 +1062,7 @@ export abstract class GuildChannel extends Channel {
   public setName(name: string, reason?: string): Promise<this>;
   public setParent(channel: CategoryChannelResolvable | null, options?: SetParentOptions): Promise<this>;
   public setPosition(position: number, options?: SetChannelPositionOptions): Promise<this>;
-  public isText(): this is TextChannel | NewsChannel;
+  public isTextBased(): this is TextChannel | NewsChannel | ThreadChannel;
```

Comment:
I'm wondering if it's better to have this as the intersection `GuildBasedChannel & TextBasedChannel` instead? It makes it a lil' more dynamic then, right?

Reply:
Absolutely, will do